polaris/polaris/management/commands/process_pending_deposits.py

```diff
@@ -312,65 +499,39 @@ def get_pending_trust_transactions():
         support claimable balances, Polaris places the transaction in the
         `pending_trust` status.
 
-        This function retrieves those transactions and ensure other processes
-        do not retreive the same transactions for processing by updating
-        Transaction.pending_execution_attempt to True.
-
         The returned transactions will be submitted if their destination
         accounts now have a trustline to the asset.
         """
-        with django.db.transaction.atomic():
-            transactions = list(
-                Transaction.objects.filter(
-                    kind=Transaction.KIND.deposit,
-                    status=Transaction.STATUS.pending_trust,
-                    pending_execution_attempt=False,
-                )
-                .select_related("asset")
-                .select_for_update()
-            )
-            Transaction.objects.filter(id__in=[t.id for t in transactions]).update(
-                pending_execution_attempt=True
+
+        transactions = list(
+            Transaction.objects.filter(
+                kind=Transaction.KIND.deposit, status=Transaction.STATUS.pending_trust,
             )
-            return transactions
+            .select_related("asset")
+            .select_for_update()
+        )
+        return transactions
 
     @staticmethod
-    def get_ready_multisig_deposits():
+    def get_unblocked_transactions():
         """
-        If the anchor's distribution account requires multiple signatures before
-        submitting to Stellar, Polaris generates the envelope and updates
-        Transaction.pending_signatures to True.
-
-        Polaris then expects the anchor to collect the necessary signatures and
-        set Transaction.pending_signatures back to False. This function checks if
-        any transaction is in this state and returns it for submission to the
-        Stellar Network.
-
-        Multisig transactions are therefore identified by a non-null envelope_xdr
-        column and a 'pending_anchor' status. The status check is important
-        because all successfully submitted transactions have their envelope_xdr
-        column set after submission and status set to 'completed'.
+        Return transactions that have been put in a SUBMISSION_STATUS.unblocked
+        state.
         """
         with django.db.transaction.atomic():
-            multisig_transactions = list(
+            unblocked_transactions = list(
                 Transaction.objects.filter(
                     kind=Transaction.KIND.deposit,
-                    status=Transaction.STATUS.pending_anchor,
+                    submission_status=Transaction.SUBMISSION_STATUS.unblocked,
                     pending_signatures=False,
-                    envelope_xdr__isnull=False,
-                    pending_execution_attempt=False,
+                    envelope_xdr__isnull=True,
```

Comment:
❓ I don't think we should have this condition, otherwise multisig transactions will never be brought back into processing right?